### ISSUE TITLE
Unable to obtain pre training model file

### ISSUE THREAD

**Issue by fzuycy** [SamVadidar/RGBT#4]
Your work is great,！But I found that I need to use the scaled yolov4-csp pre-training model file when training the model, and the model file download link given in the scaled yolov4 open source project is invalid, can you provide the pre-trained model file of scaled yolov4-csp to me? Thank u!

**Comment by SamVadidar** [comment]
I can do that in two weeks. Currently I am in holidays and do not have access to the files.

**Comment by fzuycy** [comment]
Okay, please!

I also want to ask a question, as you mentioned in your paper, as follows:
“In the channel attention by giving more weight to more uncertain feature maps (assuming each feature map is represented by a random variable), the network is signaled to extract more information by prioritizing the gradient direction in backpropagation. Therefore, the network is forced to gain more information (i.e. kernel activation) from different kernels. As a matter of fact, the proposed attention module operating only in the channel domain, showed a relatively better performance than a CBAM including both spatial andchannel attention.”
“The more the entropy of a random process, the more uncertain we are about the outcome. In the case of the attention module in the spatial domain, each grid pattern can contribute to detecting an object. As the training process goes on, the grids, which refer to an object will have lower entropy (i.e. lower uncertainty). Hence, if we use the entropy values directly, more weight will be given to grids with high entropy, which would be the background information in this case. Thus, more weights are intended to be allocated for the grids with less entropy values through the feature maps.”

Why do you focus more on channels with higher entropy in the channel attention mechanism, while in the spatial attention mechanism, you instead focus more on areas with lower entropy？

I'm looking forward to your reply！

**Comment by SamVadidar** [comment]
Let's have back propagation in mind. Along channels you want to extract information for the filters, which are not performing well (highly uncertainty). However, in spatial domain you are not interessted on back ground information to detect an object. Therefore you stay with those weights which are already quite certain (learned weights from pretrained networks, which are responsible for detecting the objects) and tune them further. Could you follow?
